javaee/src/main/java/by/teachmeskills/lesson34and35/dao/UrlPairDao.java

```diff
@@ -0,0 +1,66 @@
+package by.teachmeskills.lesson34and35.dao;
+
+import by.teachmeskills.lesson34and35.db.Connector;
+import by.teachmeskills.lesson34and35.domain.UrlPair;
+import lombok.extern.slf4j.Slf4j;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Optional;
+
+@Slf4j
+public class UrlPairDao {
+
+    private final Connection connection = Connector.getConnection();
+
+    public UrlPairDao() {
+        try {
+            createUrlTable();
+            log.info("Url pair table existing");
+        } catch (SQLException e) {
+            log.error("Something wrong with creating of url pair table");
```

Comment:
log.error("Something wrong with creating of url pair table") ->  log.error("Something wrong with creating of url pair table", e)